### DISCUSSION TITLE
Borg Log details

### DISCUSSION THREAD

**Discussion by nfrawley93** [nextcloud/all-in-one#1564] [Questions] [answered]
How do I change the level of detail shown in BorgBackup? I used to get a really nice big ol' log with file by file output of the backup process... or even at least some information. At some point though that changed and now all I get is a "Starting Backup" then nothing until "Backup Complete" along with the niceties that are shown at completion which are nice! Did I somehow change a log-level? Anyway to get all that detail back?

I've had a few issues with Borg in the past so a higher loh-level would put me at ease at least knowing it is working.

![Screenshot 2022-12-20 225154](https://user-images.githubusercontent.com/10273064/208824545-5ad677ae-bab9-4bd3-8bb2-2af6e1772ba9.jpeg)

**Comment by szaimen** [comment]
Hi, usually you should see something like this in the logs. Is this not enough?

```
Performing backup...
Starting the backup...
------------------------------------------------------------------------------
Repository: /mnt/borgbackup/borg
Archive name: 20221221_040044-nextcloud-aio
Archive fingerprint: 2a34d7aa36cd73a82d7885849f135057aa65d57900c68fe288850821d849069f
Time (start): Wed, 2022-12-21 04:00:44
Time (end):   Wed, 2022-12-21 04:00:55
Duration: 10.15 seconds
Number of files: 29416
Utilization of max. archive size: 0%
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:                1.36 GB            654.31 MB              6.57 MB
All archives:               29.27 GB             14.09 GB              1.44 GB

                       Unique chunks         Total chunks
Chunk index:                   46962               654577
------------------------------------------------------------------------------
Pruning the archives...
Warning: "--prefix" has been deprecated. Use "--glob-archives 'yourprefix*'" (-a) instead.
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
Deleted data:                    0 B                  0 B                  0 B
All archives:               29.27 GB             14.09 GB              1.44 GB

                       Unique chunks         Total chunks
Chunk index:                   46962               654577
------------------------------------------------------------------------------
Compacting the archives...
Backup finished successfully on 21.12.2022 - 04:00:59 (00 hours 00 minutes 15 seconds)
```

I am asking because the progress option was intentionally disabled as it produced some unwanted strange-looking log lines.

**Reply by nfrawley93** [reply]
That is the output after a full run. My only issue is that there isn't really a clear indicator of current progression other than "Starting the backup"
Sometimes the backup might run for 5 minutes, sometimes the backup might need to run for much longer (as much as a few hours). I know Borgmatic has a few different log-types when running, and no doubt that they likely produce some weird log lines.

Is there any middle ground for being able to view some indication of progress? It can be a bit nerve racking as I have had some odd failures with Borgmatic in the past due to power outages where once after a reboot mid backup it sat on the "Starting the backup" line for 2 days. Which is a lot given a full clean backup for my amount of data is around 4hrs.

**Comment by szaimen** [comment, marked as answer]
I'll add the progress feature back with https://github.com/nextcloud/all-in-one/pull/1622

**Reply by szaimen** [reply]
This is now released with v4.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
